9.secretstore/components/pubsub.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
We don't need this file as we are not running in standalone mode.

Reply:
Done